packages/clerk-js/src/ui/common/Billing/PlanAndBillingScreen.tsx

```diff
@@ -52,11 +63,20 @@ export const PaymentMethodSection = () => {
         {currentPlan.paymentMethod ? (
           <>
             <Box>
-              <Text>•••• {currentPlan.paymentMethod.card.last4}</Text>
-              <Text sx={t => ({ color: t.colors.$colorTextSecondary, fontSize: t.fontSizes.$sm })}>
-                {t(localizationKeys('billing.paymentMethodSection.expires'))} {currentPlan.paymentMethod.card.expMonth}/
-                {currentPlan.paymentMethod.card.expYear}
-              </Text>
+              <Flex gap={1}>
+                <CardIcon cardType={currentPlan.paymentMethod.card.brand} />
+                <Text>•••• {currentPlan.paymentMethod.card.last4}</Text>
```

Comment:
Are we able to dial in the alignment here? 

![Screenshot 2024-04-05 at 10 24 51 AM](https://github.com/clerk/javascript/assets/825855/66a3c788-c1a1-4f60-a8e0-ef608c1b4724)

Reply:
@alexcarpenter sure, thanks for your comment 🙏

Reply:
@alexcarpenter Fixed